src/main/java/com/czertainly/api/interfaces/core/web/CertificateController.java

```diff
@@ -206,4 +209,12 @@ List<CertificateContentDto> getCertificateContent(@io.swagger.v3.oas.annotations
     CertificateDetailDto createCsr(
             @RequestBody ClientCertificateRequestDto request
     ) throws ValidationException, NotFoundException, CertificateException, IOException, NoSuchAlgorithmException, InvalidKeyException, NoSuchProviderException;
+
+
+    @Operation(summary = "List Certificates Approvals")
+    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "List of all approvals for the certificate")})
+    @RequestMapping(method = RequestMethod.GET, path = "/{uuid}/approvals", produces = {"application/json"})
+    ApprovalResponseDto listOfApprovals(@Parameter(description = "Authority Instance UUID") @PathVariable String uuid,
```

Comment:
- rename to listCertificateApprovals
- parameter description Authority Instance -> Certificate

Reply:
will be fixed in next PR